impress-org/givewp
# Feature: Migrate custom fields from Form Field Manager

Depends on https://github.com/impress-org/givewp/pull/6997 and https://github.com/impress-org/give-form-field-manager/pull/412

## Description

In this pull request, we are adding a new step to migrate custom fields that were added via FFM to v2 forms. These fields will be converted into v3 blocks.

In addition to migrating the main attributes of each field, this migration also includes support for Conditional Logic (previously known as Field Visibility). We are aiming to place the fields in approximately the same location as before. This also includes support for the custom Section field, where a new section will be added and the following fields will be placed within it.

## Affects

v2 to v3 migration

## Visuals
https://github.com/impress-org/givewp/assets/3921017/f332d04b-344d-4372-ab6c-1657572f40a0

## Testing Instructions

1. Create a new v2 form
2. Add custom fields
3. Migrate that form
4. Confirm all your fields were migrated

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205643270356568
